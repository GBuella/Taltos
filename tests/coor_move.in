force
e2e4
printfen
e7e5
printfen
f1c4
printfen
f1c4
d7d6
printfen
g1f3
printfen
c8g8
c8g7
c8g6
c8g5
c8g3
c8g2
c8g1
c8g4
printfen
e1c1
e1g1
printfen
setboard rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
h2h3
printfen
undo
g2g3
printfen
undo
f2f3
printfen
undo
e2e3
printfen
undo
d2d3
printfen
undo
c2c3
printfen
undo
b2b3
printfen
undo
a2a3
printfen
undo
h2h4
printfen
undo
g2g4
printfen
undo
f2f4
printfen
undo
e2e4
printfen
undo
d2d4
printfen
undo
c2c4
printfen
undo
b2b4
printfen
undo
a2a4
printfen
undo
g1h3
printfen
undo
g1f3
printfen
undo
b1c3
printfen
undo
b1a3
printfen
undo
setboard r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R3K2R w KQkq - 0 1
d5e6
printfen
undo
g2h3
printfen
undo
d5d6
printfen
undo
g2g3
printfen
undo
b2b3
printfen
undo
a2a3
printfen
undo
g2g4
printfen
undo
a2a4
printfen
undo
e5f7
printfen
undo
e5d7
printfen
undo
e5g6
printfen
undo
e5c6
printfen
undo
e5g4
printfen
undo
e5c4
printfen
undo
e5d3
printfen
undo
c3b5
printfen
undo
c3a4
printfen
undo
c3d1
printfen
undo
c3b1
printfen
undo
f3f6
printfen
undo
f3f5
printfen
undo
f3f4
printfen
undo
f3h3
printfen
undo
f3g3
printfen
undo
f3e3
printfen
undo
f3d3
printfen
undo
h1g1
printfen
undo
h1f1
printfen
undo
a1d1
printfen
undo
a1c1
printfen
undo
a1b1
printfen
undo
f3h5
printfen
undo
f3g4
printfen
undo
e2a6
printfen
undo
e2b5
printfen
undo
e2c4
printfen
undo
e2d3
printfen
undo
e2f1
printfen
undo
e2d1
printfen
undo
d2h6
printfen
undo
d2g5
printfen
undo
d2f4
printfen
undo
d2e3
printfen
undo
d2c1
printfen
undo
e1c1
printfen
undo
e1g1
printfen
undo
e1f1
printfen
undo
e1d1
printfen
setboard r3k2r/8/8/8/8/8/8/2R1K2R w Kkq - 0 1
h1h8
printfen
undo
h1h7
printfen
undo
h1h6
printfen
undo
h1h5
printfen
undo
h1h4
printfen
undo
h1h3
printfen
undo
h1h2
printfen
undo
h1g1
printfen
undo
h1f1
printfen
undo
c1c8
printfen
undo
c1c7
printfen
undo
c1c6
printfen
undo
c1c5
printfen
undo
c1c4
printfen
undo
c1c3
printfen
undo
c1c2
printfen
undo
c1d1
printfen
undo
c1b1
printfen
undo
c1a1
printfen
undo
e1g1
printfen
undo
e1f2
printfen
undo
e1e2
printfen
undo
e1d2
printfen
undo
e1f1
printfen
undo
e1d1
printfen
