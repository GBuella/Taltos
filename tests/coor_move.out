rnbqkbnr/pppppppp/8/8/4P3/8/PPPP1PPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppp1ppp/8/4p3/4P3/8/PPPP1PPP/RNBQKBNR w KQkq - 0 2
rnbqkbnr/pppp1ppp/8/4p3/2B1P3/8/PPPP1PPP/RNBQK1NR b KQkq - 1 2
rnbqkbnr/ppp2ppp/3p4/4p3/2B1P3/8/PPPP1PPP/RNBQK1NR w KQkq - 0 3
rnbqkbnr/ppp2ppp/3p4/4p3/2B1P3/5N2/PPPP1PPP/RNBQK2R b KQkq - 1 3
rn1qkbnr/ppp2ppp/3p4/4p3/2B1P1b1/5N2/PPPP1PPP/RNBQK2R w KQkq - 2 4
rn1qkbnr/ppp2ppp/3p4/4p3/2B1P1b1/5N2/PPPP1PPP/RNBQ1RK1 b kq - 0 4
rnbqkbnr/pppppppp/8/8/8/7P/PPPPPPP1/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/6P1/PPPPPP1P/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/5P2/PPPPP1PP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/4P3/PPPP1PPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/3P4/PPP1PPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/2P5/PP1PPPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/1P6/P1PPPPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/P7/1PPPPPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/7P/8/PPPPPPP1/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/6P1/8/PPPPPP1P/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/5P2/8/PPPPP1PP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/4P3/8/PPPP1PPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/3P4/8/PPP1PPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/2P5/8/PP1PPPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/1P6/8/P1PPPPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/P7/8/1PPPPPPP/RNBQKBNR b KQkq - 0 1
rnbqkbnr/pppppppp/8/8/8/7N/PPPPPPPP/RNBQKB1R b KQkq - 1 1
rnbqkbnr/pppppppp/8/8/8/5N2/PPPPPPPP/RNBQKB1R b KQkq - 1 1
rnbqkbnr/pppppppp/8/8/8/2N5/PPPPPPPP/R1BQKBNR b KQkq - 1 1
rnbqkbnr/pppppppp/8/8/8/N7/PPPPPPPP/R1BQKBNR b KQkq - 1 1
r3k2r/p1ppqpb1/bn2Pnp1/4N3/1p2P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1P/PPPBBP1P/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn1Ppnp1/4N3/1p2P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2QPp/PPPBBP1P/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/1PN2Q1p/P1PBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/P1N2Q1p/1PPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P1P1/2N2Q1p/PPPBBP1P/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/Pp2P3/2N2Q1p/1PPBBPPP/R3K2R b KQkq a3 0 1
r3k2r/p1ppqNb1/bn2pnp1/3P4/1p2P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1pNqpb1/bn2pnp1/3P4/1p2P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnN1/3P4/1p2P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bnN1pnp1/3P4/1p2P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3P4/1p2P1N1/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3P4/1pN1P3/2N2Q1p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3P4/1p2P3/2NN1Q1p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/1N1PN3/1p2P3/5Q1p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/Np2P3/5Q1p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/5Q1p/PPPBBPPP/R2NK2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/5Q1p/PPPBBPPP/RN2K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pQp1/3PN3/1p2P3/2N4p/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PNQ2/1p2P3/2N4p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2PQ2/2N4p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N4Q/PPPBBPPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N3Qp/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N1Q2p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2NQ3p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R3K1R1 b Qkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R3KR2 b Qkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/3RK2R b Kkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/2R1K2R b Kkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/1R2K2R b Kkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN2Q/1p2P3/2N4p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P1Q1/2N4p/PPPBBPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/Bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPB1PPP/R3K2R b KQkq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/1B1PN3/1p2P3/2N2Q1p/PPPB1PPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1pB1P3/2N2Q1p/PPPB1PPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2NB1Q1p/PPPB1PPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPB1PPP/R3KB1R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPB1PPP/R2BK2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnpB/3PN3/1p2P3/2N2Q1p/PPP1BPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN1B1/1p2P3/2N2Q1p/PPP1BPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2PB2/2N2Q1p/PPP1BPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N1BQ1p/PPP1BPPP/R3K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPP1BPPP/R1B1K2R b KQkq - 1 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/2KR3R b kq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R4RK1 b kq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R4K1R b kq - 0 1
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R2K3R b kq - 0 1
r3k2R/8/8/8/8/8/8/2R1K3 b q - 0 1
r3k2r/7R/8/8/8/8/8/2R1K3 b kq - 0 1
r3k2r/8/7R/8/8/8/8/2R1K3 b kq - 0 1
r3k2r/8/8/7R/8/8/8/2R1K3 b kq - 0 1
r3k2r/8/8/8/7R/8/8/2R1K3 b kq - 0 1
r3k2r/8/8/8/8/7R/8/2R1K3 b kq - 0 1
r3k2r/8/8/8/8/8/7R/2R1K3 b kq - 0 1
r3k2r/8/8/8/8/8/8/2R1K1R1 b kq - 0 1
r3k2r/8/8/8/8/8/8/2R1KR2 b kq - 0 1
r1R1k2r/8/8/8/8/8/8/4K2R b Kkq - 1 1
r3k2r/2R5/8/8/8/8/8/4K2R b Kkq - 1 1
r3k2r/8/2R5/8/8/8/8/4K2R b Kkq - 1 1
r3k2r/8/8/2R5/8/8/8/4K2R b Kkq - 1 1
r3k2r/8/8/8/2R5/8/8/4K2R b Kkq - 1 1
r3k2r/8/8/8/8/2R5/8/4K2R b Kkq - 1 1
r3k2r/8/8/8/8/8/2R5/4K2R b Kkq - 1 1
r3k2r/8/8/8/8/8/8/3RK2R b Kkq - 1 1
r3k2r/8/8/8/8/8/8/1R2K2R b Kkq - 1 1
r3k2r/8/8/8/8/8/8/R3K2R b Kkq - 1 1
r3k2r/8/8/8/8/8/8/2R2RK1 b kq - 0 1
r3k2r/8/8/8/8/8/5K2/2R4R b kq - 0 1
r3k2r/8/8/8/8/8/4K3/2R4R b kq - 0 1
r3k2r/8/8/8/8/8/3K4/2R4R b kq - 0 1
r3k2r/8/8/8/8/8/8/2R2K1R b kq - 0 1
r3k2r/8/8/8/8/8/8/2RK3R b kq - 0 1
